spec/features/collection_drafts/forms/spatial_information_form_spec.rb

```diff
@@ -122,6 +122,21 @@
         expect(page).to have_content('Collection Draft Updated Successfully!')
       end
 
+      it 'contains the correct dropdown options' do
+        within '#draft_tiling_identification_systems_0_tiling_identification_system_name' do
+          expect(page).to have_css('option', count: 10)
```

Comment:
There is an assertion for select that has an `options` option, which would be more efficient

Reply:
https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FMatchers:has_select%3F

Reply:
Yeah that's much cleaner